kilkari/src/main/java/org/motechproject/nms/kilkari/domain/SubscriptionPack.java

```diff
@@ -13,21 +14,26 @@
 public class SubscriptionPack {
 
     @Field
+    @Unique
     private String name;
```

Comment:
Can this be null?  Also do you know the size constraints?  If so please add those annotation.